spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -65,7 +71,7 @@ public ConsumerDestination provisionConsumerDestination(String name, String grou
 				this.pubSubAdmin.createSubscription(subscription, name);
```

Comment:
minor nit: seems like the tabs are off in this block

Reply:
Ah nevermind, some of the code was compressed by the UI, I missed that there were lines between `if (this.pubSubAdmin.getTopic(name) == null) {` and `this.pubSubAdmin.createSubscription(subscription, name);`